src/rules/noStringThrowRule.ts

```diff
@@ -71,7 +71,7 @@ function walk(ctx: Lint.WalkContext<void>): void {
     return ts.forEachChild(ctx.sourceFile, function cb(node: ts.Node): void {
         if (isThrowStatement(node)) {
             const { expression } = node;
-            if (isString(expression)) {
+            if (expression !== undefined && isString(expression)) {
```

Comment:
`ThrowStatement` is possibly `undefined` now.
https://github.com/Microsoft/TypeScript/blob/v3.1.6/lib/typescript.d.ts#L1312